nav2_navfn_planner/src/navfn_planner.cpp

```diff
@@ -320,7 +320,7 @@ NavfnPlanner::getPlanFromPotential(
 
   planner_->setStart(map_goal);
 
-  int path_len = planner_->calcPath(costmap_->getSizeInCellsX() * 4);
+  int path_len = planner_->calcPath((costmap_->getSizeInCellsX() >= costmap_->getSizeInCellsY()) ? (costmap_->getSizeInCellsX() * 4) : (costmap_->getSizeInCellsY() * 4));
```

Comment:
Please break that down onto another line and use the variable here, this line has too much going on in it to easily track.